resources/templates/README.md

```diff
@@ -11,7 +13,7 @@ Description of the project
 
 # Run it locally
 
-# ...
+#
```

Comment:
Should we remove this completely or add another title?

Reply:
I've updated the PR to include the rule we discussed (`no-trailing-punctuation: false`)